p2p/metrics.go

```diff
@@ -87,13 +87,13 @@ func PrometheusMetrics(namespace string, labelsAndValues ...string) *Metrics {
 			Subsystem: MetricsSubsystem,
 			Name:      "message_receive_bytes_total",
 			Help:      "Number of bytes of each message type received.",
-		}, append(labels, "message_type")).With(labelsAndValues...),
+		}, append(labels, "message_type", "chID")).With(labelsAndValues...),
```

Comment:
[no change needed][for reviewers] "chID" appears already used many times in this repo. I would've preferred the longer and more precise "channelID" but I don't think it's worth changing b/c we want to minimize the diff to CometBFT.

Reply:
> [no change needed][for reviewers] "chID" appears already used many times in this repo. I would've preferred the longer and more precise "channelID" 

Agree
> but I don't think it's worth changing b/c we want to minimize the diff to CometBFT.

Agree